src/main/webapp/navbar.html

```diff
@@ -1,11 +1,14 @@
+<!DOCTYPE html>
 <html>
   <head>
     <script src="https://unpkg.com/vue"></script>
     <script src="https://unpkg.com/http-vue-loader"></script>
   </head>
-  <body>
+  <body style="margin: 0 auto">
     <div id="my-app">
-      <navigation :user="user"></navigation>
+      <div style="height:80px">
+        <navigation :user="user"></navigation>
+      </div>
```

Comment:
Looks like creating a css reset or normalize stylesheet that we include in all of our pages might be a more sustainable solution. Example: https://meyerweb.com/eric/tools/css/reset/
I'll look into creating one and adding it to template.html